lib/bitstyles_phoenix/components/button.ex

```diff
@@ -59,6 +65,14 @@ defmodule BitstylesPhoenix.Button do
       ~s(<a class="a-button" href="/foo">Save</a>)
   """)
 
+  story("Button with a custom link function", """
+      iex> defmodule CustomLink do
+      ...>   def link(text, opts), do: Phoenix.HTML.Tag.content_tag(:a, text, href: opts[:to])
```

Comment:
```suggestion
      ...>   def link(text, opts), do: Phoenix.HTML.Tag.content_tag(:a, text, href: opts[:to], class:  opts[:class)
```
let's at least consume the classes, since otherwise we don't see what `ui_button` even does.